src/common/monitoring/resourcemonitor.cpp

```diff
@@ -62,7 +62,7 @@ Error ResourceMonitor::Start()
     auto nodeConfig = MakeUnique<sm::resourcemanager::NodeConfig>(&mAllocator);
 
     if (auto err = mResourceManager->GetNodeConfig(nodeConfig->mNodeConfig); !err.IsNone()) {
-        return AOS_ERROR_WRAP(err);
+        LOG_ERR() << "Get node config failed, err=" << AOS_ERROR_WRAP(err);
```

Comment:
```suggestion
        LOG_WRN() << "Get node config failed, err=" << AOS_ERROR_WRAP(err);
```

Looks like the common practice is to log warning on errors, that are not returned to the caller. (I might be mistaken)

Reply:
I remember I decreased levels for certain logs to fix "multiple alerts after restart" issue.
But I haven't heard of the mentioned tendency.